### DISCUSSION TITLE
Security libraries

### DISCUSSION THREAD

**Discussion by mochiruntime** [clicepfl/clic-website#243] [Ideas]
Add and configure security libraries that add defenses out of the box ;
- HTTP headers : https://helmetjs.github.io/
- XSS sanitizer ; https://github.com/cure53/DOMPurify
- Rate limiting: https://github.com/animir/node-rate-limiter-flexible

More Express security best practices: https://expressjs.com/en/advanced/best-practice-security.html

**Comment by jonscanzi** [comment]
Might need to also add some of these protections to the nginx proxy
